src/components/AddressSearch.js

```diff
@@ -184,7 +184,7 @@ const AddressSearch = (props) => {
                     query={query}
                     requestUrl={{
                         useOnPlatform: 'all',
-                        url: `${CONFIG.EXPENSIFY.URL_API_ROOT}api?command=Proxy_GooglePlaces&proxyUrl=`,
+                        url: `${CONFIG.EXPENSIFY.PRODUCTION_API_ROOT}api?command=Proxy_GooglePlaces&proxyUrl=`,
```

Comment:
Should we use `ApiUtils.getRootApi` here or is this intentionally meant to always use the PROD api? 
_(in that case the rename makes it clear)_